example/particles/particles.hpp

```diff
@@ -28,11 +28,18 @@ namespace particles_example {
 
 typedef Kokkos::Random_XorShift64_Pool<> RNGPool;
 
-class ParticleDriver : public MultiStageDriver {
+class ParticleDriver : public EvolutionDriver {
  public:
   ParticleDriver(ParameterInput *pin, ApplicationInput *app_in, Mesh *pm)
-      : MultiStageDriver(pin, app_in, pm) {}
-  TaskCollection MakeTaskCollection(BlockList_t &blocks, int stage);
+      : EvolutionDriver(pin, app_in, pm),
+        integrator(std::make_unique<StagedIntegrator>(pin)) {}
+  TaskCollection MakeParticlesCreationTaskCollection() const;
+  TaskCollection MakeParticlesUpdateTaskCollection() const;
+  TaskCollection MakeFinalizationTaskCollection() const;
+  TaskListStatus Step();
+
+ private:
+  std::unique_ptr<StagedIntegrator> integrator;
```

Comment:
Does this really need to be a unique_ptr, why can't it just be a StagedIntegrator?

Reply:
Yeah this can just be `StagedIntegrator`, fixed